kanidmd/src/lib/be/dbvalue.rs

```diff
@@ -15,6 +19,10 @@ pub enum DbPasswordV1 {
     SSHA512(Vec<u8>, Vec<u8>),
 }
 
+//? This type is basically an alias for `TotpAlgo`.
+//? Why don't we do
+//? `type DbTotpAlgoV1 = TotpAlgo` and then
+//? derive `Serialize`/`Deserialize` on `TotpAlgo`?
```

Comment:
Because we may need to change the on disk format - it doesn't matter if they are equivalent, we need them seperate so we have the flexibility to change these in the future if they diverge and we need to perform transformations.